cms/envs/common.py

```diff
@@ -96,6 +96,8 @@
     SUPPORT_SITE_LINK,
 
     CONTACT_EMAIL,
+
+    DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH,
```

Comment:
Do we need this here? There are not any flows in Studio for which we would disable account activation right?